### PR TITLE
Initial version is ready

### PR DESCRIPTION
**Pull request by moghaddam**
The very first version of the plugin is now ready. I've not uploaded it to IntelliJ Plugin Marketplace. We need to have an organization account for that. You can run an IntelliJ instance bundled with this plugin using `./gradlew runIde`.

### CODE REVIEW

File: src/main/java/org/microshed/intellij/wizard/MicroProfileSelectionStep.java [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+        topPanelLayoutConstraint.gridy++;
+        topPanel.add(new LabeledComponent("MicroProfile Versions", mpVersionsComboBox), topPanelLayoutConstraint);
+
+
+        mpServersComboBox = new ComboBox<String>(specMatrix.getConfigs().get(mpVersionsComboBox.getItemAt(mpVersionsComboBox.getSelectedIndex()))
+                .getSupportedServers().toArray(new String[0]));
+        mpServersComboBox.setRenderer(new MPServerComboBoxRenderer());
+
+        topPanelLayoutConstraint.gridy++;
+        topPanel.add(new LabeledComponent("MicroProfile Servers", mpServersComboBox), topPanelLayoutConstraint);
```
Comment (Emily-Jiang) [inline src/main/java/org/microshed/intellij/wizard/MicroProfileSelectionStep.java]: Should say "MicroProfile Implemenations"

**Review by Emily-Jiang — COMMENTED** [review]
Is it possible not to check in the jar gradle/wrapper/gradle-wrapper.jar?
Did you check the vscode extension to see whether the two extensions sync up?

**Comment by Emily-Jiang** [comment]
@YK, can you review to see whether intellij extension and vscode extension work similar? I think it is better if two extensions UIs are similar.

**Comment by yk** [comment]
I think you got me confused with another user :)

File: src/main/java/org/microshed/intellij/wizard/MicroProfileSelectionStep.java (hunk 1 above)
Comment (moghaddam) [reply src/main/java/org/microshed/intellij/wizard/MicroProfileSelectionStep.java]: Fixed.

**Comment by moghaddam** [comment]
> Is it possible not to check in the jar gradle/wrapper/gradle-wrapper.jar?

Yes, we can remove the wrapper, but then we need to have a version of gradle installed on a computer to start the build process. The wrapper ensures that we'll all use the same gradle version to build. I just pushed a commit that removes the wrapper completely and updated the documentation for the build process.

> Did you check the vscode extension to see whether the two extensions sync up?

Yes. They have the same flow, the only item that I've not added yet is a checkbox to select/deselect all specs.

**Comment by yeekangc** [comment]
Sorry, @yk.  I believe @Emily-Jiang wanted to reach me.  Emily, I will take a look.

@moghaddam, thank you for the drop.

**Review by yeekangc — APPROVED** [review]

**Comment by yeekangc** [comment]
Merging as initial drop.  Thank you, @moghaddam.

**Pull request merged**